packages/turf-boolean/index.js

```diff
@@ -0,0 +1,21 @@
+var contains = require('./contains');
+var equals = require('./equals');
+var within = require('./within');
+
+/**
+* Contains returns true if the second geometry is completely contained by the first geometry.
+* The contains predicate returns the exact opposite result of the within predicate.
+*
+* @name boolean
+* @param {feature1} feature1
+* @param {feature2} feature2
+* @returns {Boolean}
+* @example
+* var along = turf.contains(line, point);
+*/
+
+module.exports = {
```

Comment:
Agree, that's why I suggested to have all the `boolean` modules within their own unique repos. I've only converted `@turf/boolean-contains` so far from this PR.

@rowanwins Lets stick with 1 PR per `boolean` module & all within their own repos at first, `@turf/boolean` can group them all up afterwards.